setup.py

```diff
@@ -33,7 +33,6 @@
     setup_requires=pytest_runner,
     install_requires=[
         'requests>=2.11,<3.0',
-        'urllib3>=1.24.3,<2.0'
```

Comment:
Do we want to remove it completely or leave it to grab the latest version. Otherwise I think users would have to separately download urllib3 to use TSC.

Reply:
I believe requests will install it's own dependancies if we remove this line so we'll respect whatever it's pin is. I'll double check the commit when I added this to see.

Reply:
Yup, we used to not specify at all, and let requests manage itself -- assuming we're all good on the quotes, we should be ok now. (I wouldn't merge this until @gaoang2148 confirms all is well on the Server side though)

Reply:
Oh okay that makes sense!